lib/routes/scmp/index.ts

```diff
@@ -32,56 +30,19 @@ export const route: Route = {
 async function handler(ctx) {
     const categoryId = ctx.req.param('category_id');
     const rssUrl = `https://www.scmp.com/rss/${categoryId}/feed`;
-    const { data: response } = await got(rssUrl);
-    const $ = load(response, {
-        xmlMode: true,
-    });
-
-    const list = $('item')
-        .toArray()
-        .map((elem) => {
-            const item = $(elem);
-            const enclosure = item.find('enclosure').first();
-            const mediaContent = item.find('media\\:content').toArray()[0];
-            const thumbnail = item.find('media\\:thumbnail').toArray()[0];
-            return {
-                title: item.find('title').text(),
-                description: item.find('description').text(),
-                link: item.find('link').text().split('?utm_source')[0],
-                author: item.find('author').text(),
-                pubDate: parseDate(item.find('pubDate').text()),
-                enclosure_url: enclosure?.attr('url'),
-                enclosure_length: enclosure?.attr('length'),
-                enclosure_type: enclosure?.attr('type'),
-                media: {
-                    content: Object.keys(mediaContent.attribs).reduce((data, key) => {
-                        data[key] = mediaContent.attribs[key];
-                        return data;
-                    }, {}),
-                    thumbnail: thumbnail?.attribs
-                        ? Object.keys(thumbnail.attribs).reduce((data, attr) => {
-                              data[attr] = thumbnail.attribs[attr];
-                              return data;
-                          }, {})
-                        : undefined,
-                },
-            };
-        });
```

Comment:
This also remove the abilities to return enclosures and Media RSS.

Reply:
Aren't enclosures included in the parsed rss feed?

Reply:
Please take a look at the content of https://github.com/DIYgod/RSSHub/pull/15415#issuecomment-2084414853

Reply:
The `enclosure` field is fixable. However in terms of `media`, the upstream `rss-parser` package does not support extracting it. It seems that this is fixed in the codebase in https://github.com/rbren/rss-parser/commit/eb8a4eeea0acf074a9cdb09866f1d5c64c0df88d , but it hasn't been made into a release.

So... Wait for upstream?